TLM/TLM/UI/TextureResources.cs

```diff
@@ -42,6 +43,14 @@ public class TextureResources
         public static readonly Texture2D PedestrianModeAutomaticTexture2D;
         public static readonly Texture2D PedestrianModeManualTexture2D;
 		public static readonly IDictionary<PriorityType, Texture2D> PrioritySignTextures;
```

Comment:
Minor: Weird indenting in this file.

Reply:
In Speed Limits PR #384 this file is formatted. I am trying to avoid formatting same files in multiple PR.